src/style.css

```diff
@@ -51,5 +67,35 @@ li:nth-child(even) {
 }
 
 button {
```

Comment:
- [ ] The refresh button is not properly aligned, please kindly fix that. 

***The wireframe***
![sha2](https://user-images.githubusercontent.com/65852621/141288766-d92c368f-a4e9-434c-933d-ab0f84e42636.png)

***Your implementation***
![sha3](https://user-images.githubusercontent.com/65852621/141288928-bb5b3124-ec61-4dcd-95fb-ca99c147c248.png)